aidies-project/client/src/store/campaign.js

```diff
@@ -124,6 +115,18 @@ export const getOneCampaign = (id) => {
         }
     };
 };
+export const editCampaign = ({ campaignName, summary, story, startingPrice, closingDate, userId, charity, category, id }) => {
+    return async dispatch => {
+        await fetch(`/api/campaign/${id}/edit`, {
+            method: "put",
+            headers: {
+                "Content-Type": "Application/json"
+            },
+            body: JSON.stringify({ campaignName, summary, story, startingPrice, closingDate, userId, charity, category }),
```

Comment:
Minor spacing suggestion
```suggestion
            body: JSON.stringify({
                campaignName,
                summary,
                story,
                startingPrice,
                closingDate,
                userId,
                charity,
                category,
            }),
```